packages/manager/modules/pci/src/components/project/users/users.html

```diff
@@ -108,13 +119,40 @@
             </oui-action-menu-item>
             <oui-action-menu-item
                 data-ng-if="$ctrl.editRoles"
-                data-disabled="$ctrl.isActionDisabled()"
+                data-disabled="$ctrl.constructor.isPending($row) || $ctrl.isActionDisabled()"
                 data-on-click="$ctrl.editRoles($row)"
             >
                 <span
                     data-translate="pci_projects_project_users_roles_label"
                 ></span>
             </oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.showUserInformations"
+                data-disabled="$ctrl.constructor.isPending($row) || $ctrl.isActionDisabled()"
+                data-on-click="$ctrl.showUserInformations($row)"
+            >
+                <span
+                    data-translate="pci_projects_project_users_show_user_informations_label"
+                ></span>
+            </oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.showCert"
+                data-disabled="$ctrl.constructor.isPending($row) || $ctrl.isActionDisabled()"
+                data-on-click="$ctrl.showCert($row)"
+            >
+                <span
+                    data-translate="pci_projects_project_users_show_cert_label"
+                ></span>
+            </oui-action-menu-item>
+            <oui-action-menu-item
+                data-ng-if="$ctrl.showKey"
+                data-disabled="$ctrl.constructor.isPending($row) || $ctrl.isActionDisabled()"
```

Comment:
This line is repeated multiple times. Consider moving this to a function in controller.

Reply:
Done here: https://github.com/ovh/manager/pull/5583